routers/web/repo/issue.go

```diff
@@ -904,6 +904,10 @@ func NewIssueChooseTemplate(ctx *context.Context) {
 		return
 	}
 
+	issueConfig, err := ctx.IssueConfigFromDefaultBranch()
+	ctx.Data["IssueConfig"] = issueConfig
+	ctx.Data["IssueConfigError"] = err
```

Comment:
Why open a new key, when we can also reuse `ctx.Flash.Error()`?

Reply:
Or is that not available for issues?
If that's the case, we should probably fix it.

Reply:
`ctx.Flash.Error("Test123")` does nothing, so I'm guess it is not available here

Reply:
Now I see it. It's on the main repo page not the Issues one. Is that really the right place for this?

Reply:
From testing, ctx.Flash.Error causes a few problems. I stay with the current solution.

Reply:
Something like
```suggestion
	ctx.Data["IssueConfigError"] = err // ctx.Flash.Err makes problems here
```